Problem: upgrading to uniffi 0.21 fails

```
error[E0080]: evaluation of constant value failed
[836](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:837)
 --> /home/runner/work/defi-wallet-core-rs/defi-wallet-core-rs/target/debug/build/defi-wallet-core-common-573ee4e9d2b69761/out/common.uniffi.rs:9:1
[837](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:838)
  |
[838](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:839)
9 | uniffi::assert_compatible_version!("0.19.6"); // Please check that you depend on version 0.19.6 of the `uniffi` crate.
[839](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:840)
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempt to compute `0_usize - 1_usize`, which would overflow
[840](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:841)
  |
[841](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:842)
  = note: this error originates in the macro `uniffi::deps::static_assertions::const_assert` which comes from the expansion of the macro `uniffi::assert_compatible_version` (in Nightly builds, run with -Z macro-backtrace for more info)
[842](https://github.com/crypto-com/defi-wallet-core-rs/actions/runs/3268277983/jobs/5374495048#step:5:843)
```

--- a/common/Cargo.toml
+++ b/common/Cargo.toml
@@ -46,8 +46,8 @@ tendermint = "0.23"
 tendermint-proto = "0.23"
 tendermint-rpc = "0.23"
 thiserror = "1"
-uniffi = { version = "^0.19", optional = true }
-uniffi_macros = { version = "^0.19", optional = true }
+uniffi = { version = "^0.21", optional = true }
+uniffi_macros = { version = "^0.21", optional = true }
 url = "2"
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
@@ -73,4 +73,4 @@ tonic = { version = "0.8", default-features = false, features = ["codegen", "pro
 once_cell = "1"
 
 [build-dependencies]
-uniffi_build = { version = "^0.19", features=["builtin-bindgen"], optional = true }
+uniffi_build = { version = "^0.21", features=["builtin-bindgen"], optional = true }

